tox.ini

```diff
@@ -42,6 +42,7 @@ deps =
     -rrequirements-py3.txt
     # Extras
     Pillow
+    leveldb
```

Comment:
no, leveldb shouldn't be installed for `precise` environment; it is kind of messy